packages/ERTP/src/issuer.js

```diff
@@ -41,8 +40,13 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
   const makePurse = () => {
     const purse = harden({
       deposit: (paymentP, optAmount = undefined) => {
```

Comment:
Not that it isn't expected to be a promise, it shouldn't end in `P`

Reply:
Oh yeah. I was thinking that an unwrapped promise should be treated as  a presence, but in this case, we know that it has been returned to the originator, so it must be the actual object. Thanks.